hw5/main.c

```diff
@@ -0,0 +1,54 @@
+#include "sortList.h"
+#include <stdio.h>
+#include <stdlib.h>
+
+void printMenu() {
```

Comment:
Если метод ничего не принимает, надо вместо пустых скобок написать `void`
```suggestion
void printMenu(void) {
```